renderdoc/replay/entry_points.cpp

```diff
@@ -749,6 +872,15 @@ extern "C" RENDERDOC_API void RENDERDOC_CC RENDERDOC_StartAndroidRemoteServer(co
   if(device || device[0] == '\0')
     Android::extractDeviceIDAndIndex(device, index, deviceID);
 
+  // We should hook up versioning of the server, then re-install if the version is old or mismatched
+  // But for now, just install it, if not already present
```

Comment:
The RenderDoc version should be enough, although I haven't looked into it too hard. We would just need to add that to AndroidManifest.xml and then query it with "aapt dump badging RenderDocCmd.apk".  There may be a way to do that without pulling the APK too, again just needs some research.